primitives/state-machine/src/trie_backend_essence.rs

```diff
@@ -417,7 +417,8 @@ where
 
 	/// Return the next key in the trie i.e. the minimum key that is strictly superior to `key` in
 	/// lexicographic order.
```

Comment:
Could you please update the docs?

Reply:
Not entirely sure if this is what you wanted, but done. (:

Reply:
Yeah sorry, I should have been writing more here. Your update looks like what I wanted!